LinaEngine/src/Lina_Input.cpp

```diff
@@ -8,37 +8,74 @@ Lina_Input::Lina_Input()
 	//Add a console message about correct initialization
 	Lina_Console cons = Lina_Console();
 	cons.AddConsoleMsg("Input initialized.", Lina_Console::MsgType::Success);
+
+	//Get state of the keyboard and also store the number of the keys in the keyboard at m_Numkeys
+	Uint8* keyboard = (Uint8*)SDL_GetKeyboardState(&m_NumKeys);
```

Comment:
SDL_GetKeyboardState is supposed to be called only once at the init of the program. We should use SDL_PumpEvents to update the array reference. It is not preferred to call GetKeyboardState every frame.